portlayer/attach/messages.go

```diff
@@ -14,17 +14,100 @@
 
 package attach
 
+import "golang.org/x/crypto/ssh"
+
 // All of the messages passed over the ssh channel/global mux are (or will be)
 // defined here.
 
-const (
-	WindowChangeReq = "window-change"
-)
+type Messager interface {
```

Comment:
It seems to be just serializing/deserializing messages and storing its type. Are you planning on adding more methods that would actually send/receive messages?